themes/classic/templates/checkout/_partials/steps/personal-information.tpl

```diff
@@ -4,19 +4,19 @@
   {if $customer.is_logged && !$customer.is_guest}
 
     <p class="identity">
-      {l s='Connected as [1]%1$s %2$s[/1].'
+      {l s='Connected as [1]%1$s %2$s[/1].' d='Shop.Theme.CustomerAccount'
```

Comment:
Can you put the `d` parameter on a new line, please? Also, put the closing brace on a new line too?